### PR TITLE
ML - dashboard icons and links

### PR DESCRIPTION
**Pull request by llali**
Machine Learning extension dashboard updates:
- Renamed the extension to "Machine Learning"
- Modified some icons( still waiting for some icons)
- Added finalized links 

image

### CODE REVIEW

File: extensions/machine-learning-services/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "machine-learning-services",
+  "name": "machine-learning",
```
[the remark this answers is not in the document]
Comment (alanrenmsft) [reply extensions/machine-learning-services/package.json:2]: also the scripts

**Comment by alanrenmsft** [comment]
could you please share a screenshot?

File: extensions/machine-learning-services/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (llali) [reply extensions/machine-learning-services/package.json:2]: I renamed the folder and everything build locally but failing in the lab. I'm not sure why. any ideas?

File: build/azure-pipelines/win32/continuous-build-win32.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ steps:
   condition: and(succeeded(), ne(variables['CacheRestored'], 'true'))
 - task: 1ESLighthouseEng.PipelineArtifactCaching.SaveCacheV1.SaveCache@1
   inputs:
-    keyfile: '.yarnrc, remote/.yarnrc, **/yarn.lock, !**/node_modules/**/yarn.lock, !**/.*/**/yarn.lock, !samples/**/yarn.lock'
+    keyfile: 'build/.cachesalt .yarnrc, remote/.yarnrc, **/yarn.lock, !**/node_modules/**/yarn.lock, !**/.*/**/yarn.lock, !samples/**/yarn.lock'
```
Comment (anthonydresser) [inline build/azure-pipelines/win32/continuous-build-win32.yml:25]: You are missing a ,

**Pull request merged**